rocket_component.py

```diff
@@ -0,0 +1,102 @@
+from atmosphere import Atmosphere
+import math
+
+
+class Rocket:
+  """
+  The rocket class will calculate the forces acting upon the rocket component
+  at a given time. Units will be metric for consistency.
+  """
+
+  def __init__(self) -> None:
+
+    # Basic Properties (Falcon 9 ex.)
+    self.altitude = 0  # m
+    self.length = 70  # m
+    self.width = 4  # m
+    self.velocity = 1000  # m/s
```

Comment:
Do we want to track any acceleration metrics here?

Reply:
1. placeholder value. plus initial conditions will be defined elsewhere, probably in the simulation class. this velocity represents the velocity of the previous state, not the new velocity that will be calculated.

2. no, acceleration will be calculated as a result of the forces

Reply:
1. Right now though, as it stands, this is the velocity of the current state, not the previous.  Think of state as the present rather than past or future.

When we start our sim, our rocket isn't moving at 1000 m/s or at least I wouldn't think it is because we want to start our thrusters to make it move from it being still (velocity 0 m/s) on the launch pad.

2. sounds good.